who_is_suspicious/R/statplot.R

```diff
@@ -0,0 +1,80 @@
+library(ggstatsplot)
+library(tidyverse)
+library(clock)
+
+GasTech_df <- read_csv("data/Final.csv")
+GasTech_df$Date <- date_time_parse(GasTech_df$Date,
+                                zone = "",
+                                format = "%m/%d/%Y")
+GasTech_df$CarID <- as_factor(GasTech_df$CarID)
+# 
+# GasTech_df = data.frame(GasTech)
+
+# df <- GasTech_df %>% 
+#   group_by(Employment_Type, Category, Weekday.Weekend) %>% 
+#   summarise(total_spent = sum(Price))
+
+# ggbetweenstats(
+#   data = GasTech_df,
+#   x = Employment_Type,
+#   y = Price,
+#   title = "Distribution of CC spend across locations"
+# )
+# 
+# ggbetweenstats(
+#   data = GasTech_df,
+#   x = Category,
+#   y = Price,
+#   title = "Distribution of CC spend across locations"
+# )
+
+
+
+statplotUI <- function(id) {
+  tagList(
+    titlePanel("GPS & credit card data of GASTech employees"),    
+    sidebarLayout(        
+      sidebarPanel(            
+        selectInput(NS(id, "variable"),
+                    label = "Category to compare:",
+                    choices = c("Employment_Type", "Weekday/Weekend", "Category", "Day_of_Week", "Location"),
```

Comment:
changed choices such that it includes all of the possible columns. this will determine the different categories in employment type for the visualization and statistical analysis we want to do.